src/seedu/addressbook/AddressBook.java

```diff
@@ -14,19 +14,13 @@
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Optional;
-import java.util.Scanner;
-import java.util.Set;
+import java.util.*;
```

Comment:
Try to list imported classes explicitly as required by our Java Coding Standard.

Reply:
Thank you for your review, and I have made the relevant changes.